src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+
+import { Post } from '../../types/Post';
+
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        {post.user && (
+          <UserInfo user={post.user} />
+        )}
```

Comment:
would be cool if there is not user we'll get `'Posted anonymously'`

Reply:
all done! and made some improvements after the morning lesson